src/staticanalysis/frontend/src/Tasks.vue

```diff
@@ -52,6 +55,16 @@ export default {
             name: 'Publishable issues',
             func: t => t.data.issues_publishable && t.data.issues_publishable > 0
           }
+        ],
+        sources: [
+          {
+            name: 'Try',
+            func: t => t.data.source === 'try'
+          },
+          {
+            name: 'Phabricator',
```

Comment:
A thought: maybe we should have a different name, since in the end everything is coming from Phabricator

Reply:
Ah ! I thought that too... but could not find a relevant and simple name that would not confuse users (external to relman). `Local` ? but that would be odd in opposition to `Try `...